modules/wsl-distro.nix

```diff
@@ -177,5 +177,10 @@ with lib; {
             '';
           };
         })
+        # this option doesn't exist on older NixOS, so hack.
+        (lib.optionalAttrs (builtins.hasAttr "oomd" options.systemd) {
```

Comment:
Maybe we should just lift the minimal supported version to 22.11

Reply:
That is probably better done in a separate PR.

Reply:
That'd be #177

Reply:
That's not really dropping support for 22.05 though.